### PR TITLE
wrote the first Rails script

### PR DESCRIPTION
**Pull request by yggie**
Basic setup script for most Rails application (assumes PostgreSQL has been setup)
- installs dependencies for Rails
- installs nodejs, curl
- optionally installs phantomjs
- installs RVM and sets up the ruby version and gemset
- runs all rake tasks to setup the application

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,26 @@ setup-scripts
...
+* [nodejs](http://nodejs.org/) and [npm](https://www.npmjs.org/)
+* Options:
+ - `$REQUIRED_RUBY` - specifies the required ruby version
+ - `$GEMSET` - names the gemset to be used
+ - `$SKIP_BUNDLE` - if set, will skip bundle install
+ - `$SKIP_MIGRATIONS` - if set, will skip migrations
+ - `$WITH_PHANTOMJS` - if set, will install [phantomjs](http://phantomjs.org/)
+
+This script assumes you have PostgreSQL all set up, if you don't know how to set
+it up, please refer to [this]() guide
```
Comment (YaroSpace) [inline README.md]: link is broken, need to update the ling to Postgre setup in Readme

**Comment by YaroSpace** [comment]
The script is fine with me.  
Perhaps split it up into master script and dependants for ruby, rvm, rails, specific project setup, extra libraries?  
Then we can edit only those dependant scripts if needed, and add more scripts to master script for something new.
Maybe move the whole setup to rake?

**Comment by YaroSpace** [comment]
And give everyone a task/script to write - phantomJs and dependencies, postgre, automation tools, etc., and assign them as maintainers?

File: README.md (hunk 1 above)
Comment (yggie) [inline README.md]: thats because it hasn't been written yet :smile:

**Comment by yggie** [comment]
@YaroSpace as a first iteration, I think this is sufficient, we could definitely extend this further and separate scripts. do you think it is acceptable to merge this in as it is now?

**Pull request merged**